test/defaultCollateral/DefaultCollateral.t.sol

```diff
@@ -0,0 +1,656 @@
+// SPDX-License-Identifier: MIT
```

Comment:
We should probably adapt the test so that we don't have to run source .env for them to work.   

```bash
forge install
forge test
Failing tests:
Encountered 1 failing test in test/defaultCollateral/DefaultCollateral.t.sol:DefaultCollateralTest
[FAIL: invalid rpc url: mainnet] setUp() (gas: 0)
```

Reply:
I think it's weird to have the permit2 mainnet contract there. If possible I would take it out, and just deploy it in the setup., but lets ask them